SPECS/graphviz/graphviz.spec

```diff
@@ -51,7 +51,7 @@ BuildRequires:		python2-devel
 %endif
 BuildRequires:		python3-devel, java-devel
 BuildRequires:		pkgconfig(cairo) >= 1.1.10, pango-devel, gmp-devel, lua-devel
-BuildRequires:		gd-devel, perl-devel, swig >= 1.3.33, automake, autoconf, libtool
+BuildRequires:		gd-devel, perl-devel, perl, swig >= 1.3.33, automake, autoconf, libtool
```

Comment:
Could you lint this one? We didn't do that for specs inside `SPECS-EXTENDED` and this particular spec badly needs it.:)